src/protocols/aeternity/composables/aeTippingUrls.ts

```diff
@@ -27,13 +27,17 @@ export function useAeTippingUrls() {
 
     const { aeActiveNetworkSettings } = useAeNetworkSettings();
 
-    const [verified, graylist] = await Promise.all([
-      fetchJson(`${aeActiveNetworkSettings.value.backendUrl}/verified`),
-      fetchJson(`${aeActiveNetworkSettings.value.backendUrl}/static/wallet/graylist`),
-    ]);
+    try {
```

Comment:
just did the same in my PR #2537 😅

Reply:
Yeah, I saw this. Already merged and rebased.